templates/theme/styles/base.less

```diff
@@ -56,12 +51,12 @@
   }
 }
 
-.editor .search-results .marker .region {
+:host .search-results .marker .region {
```

Comment:
For this one, I take it I need to have `atom-text-editor .search-results .marker .region, :host ...`, right?